src/app/utils/constants.ts

```diff
@@ -58,3 +58,14 @@ export const SEND_MANY_TOKEN_TRANSFER_CONTRACT_PRINCIPAL =
 export const SWAP_SPONSOR_DISABLED_SUPPORT_URL =
   'https://support.xverse.app/hc/en-us/articles/18319388355981';
 export const SUPPORT_URL_TAB_TARGET = 'SupportURLTabTarget';
+
+export const DEFAULT_TRANSITION_OPTIONS = {
+  from: {
+    x: 24,
+    opacity: 0,
+  },
+  enter: {
+    x: 0,
+    opacity: 1,
+  },
+};
```

Comment:
nit: this wouldn't be the default for all transitions. 

it might belong better with a generic hook you could use on any screen with steps? something like (not tested): 
```
const DEFAULT_TRANSITION_OPTIONS = {
    from: {	
      x: 24,	
      opacity: 0,	
    },	
    enter: {	
      x: 0,	
      opacity: 1,	
    },	
  }
};
useStepTransitions = (state: ReactState<unknown>, options = DEFAULT_TRANSITION_OPTIONS) => 
  useTransition(currentStepIndex, options);
```